### PR TITLE
Update to latest build process to simplify build of .snupkg

### PR DESCRIPTION
**Pull request by robmen**

### CODE REVIEW

File: src/wixext/WixToolset.Util.wixext.nuspec [hunk 1]
```diff
@@ -20,6 +19,7 @@
     <file src="netstandard2.0\$id$.dll" target="tools" />
 
     <file src="ARM64\*.pdb" target="pdbs\ARM64" />
+    <file src="Win32\*.pdb" target="pdbs\Win32" />
```
Comment (rseanhall) [inline src/wixext/WixToolset.Util.wixext.nuspec:22]: I don't think there should be `Win32` and `x86` pdbs.

**Review by rseanhall — COMMENTED** [review]
I was going to comment about how this nicely cleaned up the project files, but the packing in the .csproj wasn't done and the package references in the .vcxproj wasn't done.

File: src/wixext/WixToolset.Util.wixext.nuspec (hunk 1 above)
Comment (robmen) [reply src/wixext/WixToolset.Util.wixext.nuspec:22]: I thought it was odd too but the folder was being created with .pdbs so I made sure it was added.

Comment (rseanhall) [reply src/wixext/WixToolset.Util.wixext.nuspec:22]: The .wixlib only has a bind path for x86, so the only code that will actually be used in the wild will come from the x86 folder. If I remember correctly, the Win32/x86 oddness comes from the way we're getting the .wixlib project to automatically build all architectures of the dependent native projects.

**Pull request merged**